ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/ERC20Tokens.kt

```diff
@@ -48,7 +48,7 @@ enum class ERC20Tokens(
         )
     ),
     ETH(
-        contractAddress = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+        contractAddress = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2".lowercase(),
```

Comment:
why you lowercase only ETH? you have SOL too

Reply:
i think we need to lowercase not here 
because it can damage search for example @Davran331122 please check it